cypress/integration/AdvancedSearchPage/AdvancedSearchForm.feature

```diff
@@ -0,0 +1,51 @@
+Feature: As a user, I want to be able to search for a clinical trial using advanced search form
+
+	Scenario Outline: When user visits Advanced Search page, all page items are present
+		And screen breakpoint is set to "<breakpoint>"
+		Given the user navigates to "/about-cancer/treatment/clinical-trials/search/advanced"
+		Then the page title is "Find NCI-Supported Clinical Trials"
+		And the text "NCI-supported clinical trials are those sponsored or otherwise financially supported by NCI. See our guide, Steps to Find a Clinical Trial, to learn about options for finding trials not included in NCI's collection." appears below the title
+		And "Steps to Find a Clinical Trial" link has a href "https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/trial-guide"
+		And Search tip icon is displayed and text "Search Tip: All fields are optional. Skip any items that are unknown or not applicable or try our basic search." appears
+		And "basic search" link has a href "/about-cancer/treatment/clinical-trials/search"
+		And the following delighters are displayed
+			| delighter    | href                                                       | title                              | text                                                                  |
+			| cts-livehelp | /contact                                                   | Have a question?We're here to help | Chat with us: LiveHelpCall us: 1-800-4-CANCER(1-800-422-6237)         |
```

Comment:
does spacing not break with this in the title?

Reply:
No, this is how it was being fetched.